syslog/sandboxes/heka/input/syslog_udp.lua

```diff
@@ -19,6 +19,10 @@ instruction_limit   = 0
 -- template (string) - The 'template' configuration string from rsyslog.conf
 -- see http://rsyslog-5-8-6-doc.neocities.org/rsyslog_conf_templates.html
 -- template = "<%PRI%>%TIMESTAMP% %HOSTNAME% %syslogtag:1:32%%msg:::sp-if-no-1st-sp%%msg%" -- RSYSLOG_TraditionalForwardFormat
+
+-- send_decode_failures (bool) - If true, any decode failure will inject a
```

Comment:
I think it would be better if the type reflected the error e.g. 'error.decode', 'error.closed' etc and the payload contains any additional detail